include/illixr/plugin.hpp

```diff
@@ -16,9 +17,9 @@ namespace ILLIXR {
 using plugin_id_t = std::size_t;
 
 /*
- * This gets included, but it is functionally 'private'. Hence the double-underscores.
+ * This gets included, but it is functionally 'private'. Hence, the double-underscores.
```

Comment:
Isn't this comment out-dated with the new name in the next line?

Reply:
yes, I will fix it in the merge